cmds/core/ps/ps.go

```diff
@@ -162,18 +161,35 @@ func (pT *ProcessTable) PrepareString() {
 }
 
 // For now, just read /proc/pid/stat and dump its brains.
-func ps(pT *ProcessTable, w io.Writer) error {
-	if len(pT.table) == 0 {
+func ps(args []string, w io.Writer) error {
```

Comment:
if you make this
ps(w io.Writer, args ...string)
then it is possible to call ps without any args, should we ever want that
ps(os.Stdout)
from what I can see, it won't break anything and it will make calling ps more convenient. The general rule is that things that might be variadic should be the final arg.